tests/components/fan/common.py

```diff
@@ -21,20 +23,24 @@ def turn_on(hass, entity_id: str = None, speed: str = None) -> None:
         ] if value is not None
     }
 
-    hass.services.call(DOMAIN, SERVICE_TURN_ON, data)
+    hass.async_create_task(
+        hass.services.async_call(DOMAIN, SERVICE_TURN_ON, data))
```

Comment:
Why not use `hass.services.async_call` within the tests, instead of converting this method to a callback? Those are only legacy helpers I added when I removed the service helper methods. Idealy we remove them at some point.

Reply:
That's how we're doing it for all the async conversions: https://github.com/home-assistant/home-assistant/blob/82edea60774fc1caa2d107d105ea95e65f15e4cf/tests/components/light/common.py

Reply:
Definitely not opposed to getting rid of those, but I'm not going to combine it into this refactoring.